### ISSUE TITLE
Entropy scheduling

### ISSUE THREAD

**Issue by zxzzz0** [opendilab/DI-engine#15]
- [x] I have marked all applicable categories:
    + [ ] exception-raising bug
    + [ ] RL algorithm bug
    + [ ] system worker bug
    + [ ] system utils bug
    + [ ] code design/refactor
    + [x] documentation request
    + [ ] new feature request
- [x] I have visited the [readme](https://github.com/opendilab/DI-engine/blob/github-dev/README.md) and [doc]()
- [x] I have searched through the [issue tracker](https://github.com/opendilab/DI-engine/issues) and [pr tracker](https://github.com/opendilab/DI-engine/pulls)
- [x] I have mentioned version numbers, operating system and environment, where applicable: (Not applicable)

In reinforcement learning there is well known explore-exploit dilemma. In league training it's crucial that we can have a better entropy coefficient scheduling because of the following reasons:
(1) If entropy of policy drops too fast to zero, it might get stuck in local optimum and failed to explore more states.
(2) If entropy of policy drops too slow, it might fail to select the right action at pivotal moments and the training is very slow.

One solution to address above problem is to have a good scheduling. Assume there are some validation measurements that we can use like _win rate_, we only decrease entropy coefficient when the win rate is on plateau.

It's similar to learning rate scheduler `ReduceLROnPlateau` in PyTorch [link](https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.ReduceLROnPlateau.html)

May us know if there could be some documentations about how entropy scheduling can be supported?

**Comment by PaParaZz1** [comment]
This problem can be abstracted into a general problem——how to add customized scheduling module in the whole training pipeline, and we will provide some examples in one week.

**Comment by PaParaZz1** [comment]
This issue has been solved in #38
